### PR TITLE
OMEGA-227: Get access requests related to customer id

### PR DESCRIPTION
**Pull request by ragnhildlarsen**
OMEGA-227

## What is the reason for this change?

The Access Overview table currently only shows access requests related to the person logged in (request it has sent or request it has received). If a user have access to a customer relationship/contract it should see all the other people that have access to the same customer relationship.

### What is the expected result?

Petter Pettersen should be able to see all other people that also have access to Per Hansens customer relationship, like Randi Rystad. Same for Randi, she should see Petter in the list when she looks at Per Hansen's customer relationship. Per Hansen, who is the owner of the customer relationship should see both Petter and Randi in the table.

Screenshot 2020-09-25 at 15 46 02

## How is it solved?

This is solved by using the new endpoint `/accessrequest/customer/{customerId}` instead of `/me/accessrequest/received` that was previous used. The requests are fetched with the customerId from the url when the user clicks the Tilganger tab.

## Any changes/updates that may affect others?

|                       | Changed |
| --------------------- | ------- |
| Unit tests            |        |
| Dependencies          |         |
| Environment variables |         |
| Build scripts         |         |

## How can it be tested and verified?

**The accesses described here is testable in the dev environment.**

- [ ] Petter Pettersen should be able to see all other people that also have access to Per Hansens customer relationship, like Randi Rystad.  
- [ ] Same for Randi, she should see Petter in the list when she looks at Per Hansen's customer relationship.
- [ ] Per Hansen, who is the owner of the customer relationship should see both Petter and Randi in the table.

### CODE REVIEW

File: packages/web/src/store/accessRequestReceived/accessRequestReceived.ts [hunk 1]
```diff
@@ -61,12 +63,16 @@ const accessRequestReceived = createSlice({
 })
 
 export const fetchAccessRequestReceived = (
+  customerId: string,
```
Comment (eTallang) [inline packages/web/src/store/accessRequestReceived/accessRequestReceived.ts:66]: This method should probably be renamed. E.g. to `fetchAccessRequestsByCustomerId`

File: packages/web/src/store/accessRequestReceived/accessRequestReceived.ts [hunk 2]
```diff
@@ -61,12 +63,16 @@ const accessRequestReceived = createSlice({
...
   accessToken: string
 ): AppThunk => async dispatch => {
   const { actions } = accessRequestReceived
   try {
     dispatch(actions.getAccessRequestReceivedStart())
```
Comment (eTallang) [inline packages/web/src/store/accessRequestReceived/accessRequestReceived.ts:71]: This action should probably be renamed. E.g. to `getAccessRequestsByCustomerIdStart`. Same with the success- and failure-actions.

Comment (eTallang) [reply packages/web/src/store/accessRequestReceived/accessRequestReceived.ts:71]: Hmm, or is the `ByCustomerId`-part implementation-details? Would it be possible to rename them simply `getAccessRequestsStart()` and so forth?

**Review by eTallang — CHANGES_REQUESTED** [review]

File: packages/web/src/store/accessRequestReceived/accessRequestReceived.ts (hunk 1 above)
Comment (eTallang) [reply packages/web/src/store/accessRequestReceived/accessRequestReceived.ts:66]: Following the logic of the comment below, maybe we could rename this `fetchAccessRequests`?

File: packages/web/src/store/accessRequestReceived/accessRequestReceived.ts (hunk 2 above)
Comment (ragnhildlarsen) [reply packages/web/src/store/accessRequestReceived/accessRequestReceived.ts]: renamed to `accessRequests`

File: packages/web/src/store/accessRequestReceived/accessRequestReceived.ts (hunk 1 above)
Comment (ragnhildlarsen) [reply packages/web/src/store/accessRequestReceived/accessRequestReceived.ts:66]: renamed to `accessRequests`

**Review by eTallang — APPROVED** [review]
LGTM

**Pull request merged**